WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasMultiLineEditBox.lua

```diff
@@ -286,11 +317,14 @@ local function Constructor()
 	local button = CreateFrame("Button", ("%s%dButton"):format(Type, widgetNum), frame, "UIPanelButtonTemplate")
 	button:SetPoint("BOTTOMLEFT", 0, 4)
 	button:SetHeight(22)
-	button:SetWidth(label:GetStringWidth() + 24)
+	button:SetWidth(100)
```

Comment:
did you make sure it fits in every locale with the fixed width?

Reply:
I checked german, for which it was plenty. Since I'm adding a few buttons, I want them all to have the same width.

Reply:
ok!